integration-tests/pages/userSearchResultsPage.js

```diff
@@ -11,6 +11,7 @@ const authUserSearchPage = () =>
     getPaginationResults: () => cy.get('.moj-pagination__results'),
     filter: () => cy.get('[name="status"]'),
     submitFilter: () => cy.get('button[type="submit"]'),
+    download: () => cy.get('button[id="download-button"]'),
```

Comment:
Is there something wrong with using an id rather than a data-qa attribute?

Reply:
normally we try and find attributes by other means, we don't try to add `id`s to pages if they don't need them

Reply:
i.e. if we want to allow the user to navigate to parts of the page then `id` would make sense but if it is just for testing then `data-qa` or equivalent would be better